### PR TITLE
kono front issue107 & 109

### PR DESCRIPTION
**Pull request by Sunnynah**
- change mobile header dropdown speed
- change mobile header shadow
- change grid panel default image

### CODE REVIEW

**Comment by Sunnynah** [comment]
#107 #109

File: kono-front/src/components/common/GridElementPanel.js [hunk 1]
```diff
@@ -103,9 +107,17 @@ export default ({
...
-                        : <Background2 />
+                        : (gridRow + gridColumn) % 6 == 1
+                            ? <Background2 />
+                            : (gridRow + gridColumn) % 6 == 2
+                                ? <Background3 />
+                                : (gridRow + gridColumn) % 6 == 3
+                                    ? <Background4 />
+                                    : (gridRow + gridColumn) % 6 == 4
+                                        ? <Background5 />
+                                        : <Background6 />
```
Comment (inhibitor1217) [inline kono-front/src/components/common/GridElementPanel.js]: switch-case 문으로 바꿔봅시당

Comment (Sunnynah) [reply kono-front/src/components/common/GridElementPanel.js]: 크크 바꿨습니다!

**Comment by inhibitor1217** [comment]
fix #107 and #109

**Pull request merged**